src/plone/restapi/serializer/user.py

```diff
@@ -34,6 +45,7 @@ def __call__(self):
             "id": user.id,
             "username": user.getUserName(),
             "roles": roles,
+            "can_delete": self.can_delete(roles),
```

Comment:
@davisagli analyzing your comment, I noticed that the user service uses the permissions [Plone Site Setup: Users and Groups](https://github.com/plone/plone.restapi/blob/db7d5dffe18d39d7487b484f5124b0abf48cfa9e/src/plone/restapi/services/users/get.py#L178-L184), which is from Plone, and [plone.restapi: Access Plone user information](https://github.com/plone/plone.restapi/blob/db7d5dffe18d39d7487b484f5124b0abf48cfa9e/src/plone/restapi/services/users/get.py#L186-L190), which is from `plone.restapi`. I wonder if we couldn't just use Plone's. They have the same roles now.

Reply:
This was @sneridagh' PR who created the permission: https://github.com/plone/plone.restapi/pull/525

Reply:
>  So I think we should only add the can_delete key if the current user has the plone.app.controlpanel.UsersAndGroups permission.

Done

Reply:
> I noticed that the user service uses the permissions [Plone Site Setup: Users and Groups](https://github.com/plone/plone.restapi/blob/db7d5dffe18d39d7487b484f5124b0abf48cfa9e/src/plone/restapi/services/users/get.py#L178-L184), which is from Plone, and [plone.restapi: Access Plone user information](https://github.com/plone/plone.restapi/blob/db7d5dffe18d39d7487b484f5124b0abf48cfa9e/src/plone/restapi/services/users/get.py#L186-L190), which is from plone.restapi. I wonder if we couldn't just use Plone's. They have the same roles now.

@wesleybl It could be a breaking change if someone has customized the roles for that permission. It may be worth more thought but I would leave it out of this PR.

(I don't consider the change from `Manage portal` to `Plone Site Setup: Users and Groups` to be breaking, because `Manage portal` should only ever be assigned to Managers, who should also always have `Plone Site Setup: Users and Groups`.)